doc/stages/readers.tindex.rst

```diff
@@ -43,7 +43,7 @@ merge the data.
           "type":"readers.tindex",
           "filter_srs":"+proj=lcc +lat_1=43 +lat_2=45.5 +lat_0=41.75 +lon_0=-120.5 +x_0=399999.9999999999 +y_0=0 +ellps=GRS80 +units=ft +no_defs",
           "filename":"index.sqlite",
-          "where":"location LIKE \'%nteresting.las%\'",
+          "where":"location LIKE \"%nteresting.las%\"",
```

Comment:
I think the original was fine here.

Reply:
I think so too, might have been my linter 5ec3f92